tests/console/clamav.pm

```diff
@@ -40,6 +40,17 @@ sub run {
     $self->select_serial_terminal;
 
     zypper_call('in clamav vim');
+    zypper_call('info clamav');
+
+    # Check Clamav version
+    # Jira ID SLE-16780: upgrade Clamav SLE
+    my $current_ver = script_output("zypper info clamav | grep Version | cut -d':' -f2| cut -d'-' -f1");
```

Comment:
my $current_ver = script_output("zypper info clamav | grep Version | cut -d':' -f2 | cut -d'-' -f1"); (add a blank)

Reply:
do you mean thte code tidy problem ?

Reply:
:) yes, minor issue

Reply:
good. I just run done the tidy, thanks :+1: